torch/csrc/jit/codegen/cuda/lower_alias_memory.cpp

```diff
@@ -71,19 +92,29 @@ bool isSerialBroadcastResolution(TensorView* producer, TensorView* consumer) {
   // Serial concrete loop id's that cover consumer's iter domain.
   std::vector<Val*> consumer_serial_loop_concrete_ids;
```

Comment:
nit:
```suggestion
  // Serial concrete loop id's that cover loop's iter domain.
  std::vector<Val*> serial_loop_concrete_ids;
```

Reply:
Thanks for the suggestion. I'll use the suggested name but delete the comment as it doesn't give anything other than what the variable name implies.